docs/pages/agents/introduction.mdx

```diff
@@ -29,85 +50,65 @@ services are enabled by default and how to enable a particular service.
 Agents typically run in the same private networks as the resources they proxy.
 They should be the only clients that can access a resource without Teleport.
 
-In this setup, agents dial the Teleport Proxy Service in order to establish
+In this setup, Agents dial the Teleport Proxy Service in order to establish
 reverse SSH tunnels. While the Proxy Service remains open to the public internet
-via its HTTPS port, agents require no open ports or public address.
+via its HTTPS port, Agents require no open ports or public address.
```

Comment:
```suggestion
via its HTTPS port, Agents require only outbound network access to the Proxy Service.
```
This is a little misleading... And I know it wasn't part of the original change. However, this could be clarified with this change.

Reply:
The change made it less explicit that you can run Agents behind a firewall, so I've incorporated this suggestion and added that language as well.